exonum-java-binding-core/pom.xml

```diff
@@ -214,6 +214,26 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-source-plugin</artifactId>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
I believe it’s OK at this stage, but if we want Maven to correctly track dependencies between various modules required for a packaged artefact (core, core-native, app, native-ITs, fakes, etc.), we’ll need to add the assembly definition in a separate module which sole purpose is packaging the artefact.

Reply:
Sounds interesting, not sure how to set up assmebly plugin in this case though